src/configuration.rs

```diff
@@ -22,6 +23,23 @@ pub mod action;
 pub mod action_set;
 mod action_set_index;
 
+#[derive(Deserialize, Debug, Clone)]
+pub struct ExpressionItem {
+    pub key: String,
```

Comment:
can we make `key` optional? it would default to the expression value (not the referenced value, though).

Reply:
I think it makes for a very horrible default, but... 🤷